src/app/loadout-drawer/loadout-item-conversion.ts

```diff
@@ -57,7 +57,7 @@ export function getItemsFromLoadoutItems(
         ? makeFakeItem(defs, buckets, undefined, loadoutItem.hash)
         : makeFakeD1Item(defs, buckets, loadoutItem.hash);
       if (fakeItem) {
-        fakeItem.equipped = loadoutItem.equipped;
+        fakeItem.equipped = loadoutItem.equip;
```

Comment:
This line seems pointless?

Reply:
Totally. I'll remove it when I swap to `ResolvedLoadoutItem`.